packages/core/package.json

```diff
@@ -111,7 +110,7 @@
     "normalize-path": "^3.0.0",
     "object-hash": "^3.0.0",
     "p-limit": "^2.3.0",
-    "pirates": "^4.0.1",
+    "pirates": "4.0.4",
```

Comment:
This is a "it's our fault, we were relying on unintentional behavior that broke now" thing but I don't want to deal with it in this PR